spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -465,6 +474,16 @@ def check_response_error(code)
           expect(existing_contrib.visibility).to eq 'private'
         end
       end
+
+      context 'with allcaps or mixed case strings' do
+        let(:request_data) { existing_contrib_ids.merge(visibility: 'PUBLIC', status: 'New') }
+        it 'downcases appropriately' do
+          http_request # defined in context, uses request_data
+          expect { existing_contrib.reload }.not_to change { existing_contrib.featured }
+          expect(existing_contrib.status).to eq 'new'
+          expect(existing_contrib.visibility).to eq 'public'
+        end
+      end
```

Comment:
- Maybe the POST and PATCH could use a shared_example for these specs?
- seems like testing for no changes to the `Contribution.count` is part of this case-sensitivity context?

Reply:
I don't think `Contribution.count` is important here.  That is established elsewhere.  The overlap is 2 lines (simple value checks, on different objects), so no real benefit from adding shared_examples for them.